apps/docs/content/getting-started/individual-packages.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Individual Packages
+description: An how to guide to install Hopper UI packages individually.
+order: 3
+---
+
+If you need a particluar installation, you can install Hopper UI packages individually.
+
+## Components
+
+Hopper Components are React Components, if you are using another framework, or React Native, you should use [components tokens](/tokens/getting-started/introduction) directly when developping your own components.
```

Comment:
```suggestion
Hopper Components are React Components. If you are using another framework, or React Native, you should use [components tokens](/tokens/getting-started/introduction) directly when developing your own components.
```
Feel free to update the sentence, but it doesn't seem right to me, they are two different sentences.